sipa/blueprints/usersuite.py

```diff
@@ -613,3 +616,89 @@ def reset_wifi_password():
     return render_template('generic_form.html',
                            page_title=gettext("Neues WLAN Passwort"),
                            form_args=form_args)
+
+
+@bp_usersuite.route("/request-repayment", methods=["GET", "POST"])
+@login_required
+def request_repayment():
+    """
+    Request a repayment of excess membership contributions
+    """
+
+    form = RequestRepaymentForm()
+
+    if form.validate_on_submit():
+        return redirect(
+            url_for(
+                ".request_repayment_confirm",
+                beneficiary=form.beneficiary.data,
+                iban=form.iban.data,
+                amount=form.amount.data,
+            )
+        )
+    elif form.is_submitted():
+        flash_formerrors(form)
+
+    form_args = {
+        "form": form,
+        "cancel_to": url_for(".index"),
+        "submit_text": gettext("Weiter"),
+    }
+
+    return render_template(
+        "generic_form.html",
+        page_title=gettext("Rücküberweisung beantragen"),
+        form_args=form_args,
+    )
+
+
+@bp_usersuite.route("/request-repayment/confirm", methods=["GET", "POST"])
+@login_required
+def request_repayment_confirm():
+    """
+    Request a repayment of excess membership contributions
+    """
+
+    beneficiary = request.args.get("beneficiary", None)
+    iban = request.args.get("iban", None, lambda x: IBAN(x, validate_bban=True))
+    amount = request.args.get("amount", None, lambda x: Decimal(x))
```

Comment:
with `from functols import partial` you can pass the callables directly:
```suggestion
    iban = request.args.get("iban", None, partial(IBAN, validate_bban=True))
    amount = request.args.get("amount", None, Decimal)
```